pandas/core/arrays/boolean.py

```diff
@@ -257,6 +257,8 @@ class BooleanArray(BaseMaskedArray):
 
     # The value used to fill '_data' to avoid upcasting
     _internal_fill_value = False
+    _TRUE_VALUES = {"True", "TRUE", "true", "1", "1.0"}
```

Comment:
these could also be module level (doesn't matter), note in the parser itself we only have

```
    object _true_values = [b'True', b'TRUE', b'true']
    object _false_values = [b'False', b'FALSE', b'false']

```
but yeah i think we agreed the 1/1.0 are fine.

Reply:
In this case let's keep it there